tests/integrations/anthropic/test_anthropic.py

```diff
@@ -0,0 +1,215 @@
+import pytest
+from unittest import mock
+from anthropic import Anthropic, Stream, AnthropicError
+from anthropic.types import Usage, ContentBlock, MessageDeltaUsage, TextDelta
+from anthropic.types.message import Message
+from anthropic.types.message_start_event import MessageStartEvent
+from anthropic.types.content_block_start_event import ContentBlockStartEvent
+from anthropic.types.content_block_delta_event import ContentBlockDeltaEvent
+from anthropic.types.content_block_stop_event import ContentBlockStopEvent
+from anthropic.types.message_delta_event import MessageDeltaEvent, Delta
+
+from sentry_sdk import start_transaction
+from sentry_sdk.consts import OP
+from sentry_sdk.integrations.anthropic import (
+    AnthropicIntegration,
+    TOTAL_TOKENS_USED,
+    PROMPT_TOKENS_USED,
+    COMPLETION_TOKENS_USED,
+)
+
+
+EXAMPLE_MESSAGE = Message(
+    id="id",
+    model="model",
+    role="assistant",
+    content=[ContentBlock(type="text", text="Hi, I'm Claude.")],
+    type="message",
+    usage=Usage(input_tokens=10, output_tokens=20),
+)
+
+
+@pytest.mark.parametrize(
+    "send_default_pii, include_prompts",
+    [
+        (True, True),
+        (True, False),
+        (False, True),
+        (False, False),
+    ],
+)
+def test_nonstreaming_create_message(
+    sentry_init, capture_events, send_default_pii, include_prompts
+):
+    sentry_init(
+        integrations=[AnthropicIntegration(include_prompts=include_prompts)],
+        traces_sample_rate=1.0,
+        send_default_pii=send_default_pii,
+    )
+    events = capture_events()
+    client = Anthropic(api_key="z")
+    client.messages._post = mock.Mock(return_value=EXAMPLE_MESSAGE)
+
+    messages = [
+        {
+            "role": "user",
+            "content": "Hello, Claude",
+        }
+    ]
+
+    with start_transaction(name="anthropic"):
+        response = client.messages.create(
+            max_tokens=1024, messages=messages, model="model"
+        )
+
+    assert response == EXAMPLE_MESSAGE
+    usage = response.usage
+
+    assert usage.input_tokens == 10
+    assert usage.output_tokens == 20
+
+    assert len(events) == 1
+    (event,) = events
+
+    assert event["type"] == "transaction"
+    assert event["transaction"] == "anthropic"
+
+    assert len(event["spans"]) == 1
+    (span,) = event["spans"]
+
+    assert span["op"] == OP.ANTHROPIC_MESSAGES_CREATE
+    assert span["description"] == "Anthropic messages create"
+    assert span["data"]["ai.model_id"] == "model"
+
+    if send_default_pii and include_prompts:
+        assert span["data"]["ai.input_messages"] == messages
+        assert span["data"]["ai.responses"] == [
+            {"type": "text", "text": "Hi, I'm Claude."}
+        ]
+    else:
+        assert "ai.input_messages" not in span["data"]
+        assert "ai.responses" not in span["data"]
+
+    assert span["data"][PROMPT_TOKENS_USED] == 10
```

Comment:
Some of this has changed a bit in the langchain merge as well, it's span["measurements"] I believe